lib/cucumber_factory/factory.rb

```diff
@@ -13,10 +13,11 @@ class Error < StandardError; end
     VALUE_INTEGER = /\d+/
     VALUE_DECIMAL = /[\d\.]+/
     VALUE_STRING = /"[^"]*"|'[^']*'/
+    VALUE_FILE = /<FILE:[^>]*>/
```

Comment:
```suggestion
    VALUE_FILE = /file:#{VALUE_STRING}/
```
Does it cause troubles that we are matched by the VALUE_STRING regex as well?